### DISCUSSION TITLE
Subscriptions / in-app purchases

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#14446] [General - Components / Directives / etc]
Hello,

I would like to ask you whether anyone has experience with Quasar mobile app (Cordova) & in-app purchases / subscriptions. 

Any tips, guides and recommendations would be welcome!

Thanks.

**Comment by bdelate** [comment]
Same here. Would love some more info (guides, articles, previous experience etc).

When searching for anything purchase/subscription related I notice that Ionic Framework pops up fairly regularly along with this plugin: https://github.com/j3k0/cordova-plugin-purchase

Would the above plugin be the recommended way of implementing purchasing/subscriptions in Quasar?

**Reply by i-mage** [reply]
Hello,

We have implemented RevenueCat (https://www.revenuecat.com/) Cordova SDK for our Quasar app (Breathjoy). RevenueCat docs are very helpful, we have subscription model running on production for both iOS and Android, all works as expected.

If you need any help, just let me know.

**Reply by bdelate** [reply]
Thanks @i-mage for the response, I'll take a look at RevenueCat.

Coming from a decade of building web based SaaS apps, we've always used Stripe or Paddle to manage payments and subscriptions. It's really disappointing that Apple and Google have such closed, locked down ecosystems for payments along with insanely high fees.

Anyway, I'll look into RevenueCat.

**Reply by i-mage** [reply]
We use Stripe on our websites but it is not an option for app subscriptions. 

First, we tried to implement to Apple and Google subscription by ourselves but once we have seen the docs and the complexity of such task, we reviewed several third-party vendors and decided for RevenueCat. They also have excellent support, I would like to add.
